app/models/user.rb

```diff
@@ -6,7 +6,8 @@ class User < ApplicationRecord
   validates :posts_counter, numericality: { only_integer: true, greater_than: 0 }
 
   # A method that returns the 3 most recent posts for a given user.
-  def recent_posts(user_id)
-    User.find(user_id).post.last(5)
+  def recent_posts(author_id)
+    user = User.find(author_id)
+    Post.where('author_id = ?', user.id).last(5)
```

Comment:
- [ ] Kindly update the `posts_counter` validation because currently it expects that a newly created user should already have a post which is impossible. Kindly use this instead: `greater_than_or_equal_to: 0`. See the error that occurs as a result of this when creating a user 👇 
![image](https://user-images.githubusercontent.com/46839548/189398094-e7526c71-e8bd-4b85-adc0-6abc0459305e.png)